src/cljdoc/server/pedestal.clj

```diff
@@ -372,8 +381,7 @@
    {:keys [route-name] :as route}]
   (->> (case route-name
          :home       [{:name ::home :enter #(pu/ok-html % (render-home/home))}]
-         :search     [{:name ::search :enter #(pu/ok-html % (search/search-page %))}]
-         :suggest    [{:name ::suggest :enter search/suggest-api}]
```

Comment:
I think the suggest API should be under /api because it returns JSON, not a page. Also, I think search-suggest is better then just suggest to make it clear it is in the context of search, though I am very open to discussion. Thoughts?

Reply:
Fair point. I think arguments could be made for either location but if you feel that it makes more sense under `/api/` I don't have anything to object.